### PR TITLE
Update page metadata, favicons, and manifest.json

### LINKED ISSUE #112: As a user I can tell what site I'm on in the tab title
## ✨ What new feature do you want to see?

We need "Pie for Providers" and our favicon in the tab title

## 🧶 Acceptance Criteria

I can see "Pie for Providers" and our favicon in the tab title instead of the default React app stuff

## 🏖 Implementation Details
Link to SVG: https://drive.google.com/open?id=1f7o0tODOH_G2fQYQCa67oYG9jokG9iCj please talk to @katelovescode to access the google drive link

https://create-react-app.dev/docs/title-and-meta-tags/
https://medium.com/@jenniferdobak/the-public-folder-and-favicons-in-create-react-app-8dc2cc1d492b

### PR DESCRIPTION
**Pull request by arku**
# 💅🏼 What issue does this fix?

#112

Updates the following:
- page title and description
- favicons
- manifest.json

**Screenshot**

image

## 🍂 Before You Submit

* [ ] Did you write tests?
* [ ] Did you run `bundle exec rspec` from the root?
* [ ] Did you run `bundle exec rails rswag` from the root?
* [ ] Did you run `bundle exec rubocop` from the root?
* [ ] Did you run `yarn lint` in `/client`?
* [ ] Did you run `yarn test` in `/client`?
* [ ] Are your primary keys UUIDs on any new tables?

## 🛷 Deployment Considerations

* [ ] Data Migrations
* [ ] Schema Migrations
* [ ] Dependencies

## 🧳 Steps to test

Get the app up and running and go to the login page and observe the page tab.

### CODE REVIEW

File: client/public/index.html [hunk 1]
```diff
@@ -4,12 +4,17 @@
     <meta charset="utf-8" />
     <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <meta name="theme-color" content="#000000" />
+    <meta name="theme-color" content="#ffffff">
     <meta
       name="description"
-      content="Web site created using create-react-app"
+      content="Pie for Providers uses technology to help child care providers earn more money and to help low-income families access early care and education. We are working for a more equitable early childhood field through fair pay for caregivers, effective government funding systems, and sustainable child care business models."
```
Comment (arku) [inline client/public/index.html:10]: This is just a placeholder. @csprayregen Can you update it?

Comment (csprayregen) [reply client/public/index.html:10]: What is this for @arku ?

Comment (arku) [reply client/public/index.html:10]: @csprayregen [Meta description](https://moz.com/learn/seo/meta-description).

Comment (csprayregen) [inline client/public/index.html:10]:
```suggestion
      content="Providers - manage your childcare subsidy payments with less time and hassle. Predict payments, fix errors, and claim all your revenue. "
```

Comment (csprayregen) [reply client/public/index.html:10]: @arku Let's go with the above

**Pull request merged**